analyze/action.yml

```
@@ -69,6 +69,8 @@ inputs:
     required: true
     default: "true"
   token:
+    description: "Token for accessing /code-scanning/analysis endpoints (used by `upload-database` and `upload`)"
+    required: false
```

We think this provides a bit too much detail on internals so would like to suggest:
```suggestion
    description: "GitHub token to use for authenticating with this instance of GitHub. The token needs the `security-events: write` permission."
    required: false
```